web/ui/react-app/src/ExpressionInput.test.tsx

```diff
@@ -28,8 +28,8 @@ describe('ExpressionInput', () => {
   const expressionInput = mount(<ExpressionInput {...expressionInputProps} />);
 
   it('renders a downshift component', () => {
-    const downshift = expressionInput.find(Downshift);
-    expect(downshift.prop('inputValue')).toEqual('node_cpu');
+    const downshift = expressionInput.find(Input);
+    expect(downshift.prop('value')).toEqual('node_cpu');
```

Comment:
While this makes the test pass, it no longer fits to the test goal of `it('renders a downshift component', ...)`.

Reply:
(I'm fine changing it in either direction, whichever you think makes more sense)

Reply:
Yes, will change it. One thing I notice here is the shared instance of ExpressionInput between tests. I think each test should operate over a fresh instance of the Input and tests not to depend on each other.

Reply:
Yeah, makes sense to me, even though in this instance it's probably not super important. /cc @hooten

Reply:
That sounds good to me!